packages/core/src/blitz-app-root.tsx

```diff
@@ -86,7 +78,9 @@ export function withBlitzAppRoot(UserAppRoot: React.ComponentType<any>) {
     return (
       <>
         {noPageFlicker && <NoPageFlicker />}
-        <UserAppRoot {...props} Component={withBlitzInnerWrapper(props.Component)} />
+        <UserAppRoot {...props}>
+          <BlitzInnerRoot Component={props.Component} />
+        </UserAppRoot>
```

Comment:
This change causes `<BlitzInnerRoot />` to never be rendered in the tree. Check your react dev tools and you'll see.